custom/conf/app.ini.sample

```diff
@@ -149,6 +149,8 @@ SHOW_USER_EMAIL = true
 DEFAULT_THEME = gitea
 ; All available themes. Allow users select personalized themes regardless of the value of `DEFAULT_THEME`.
 THEMES = gitea,arc-green
+; All available reactions. Allow users react with different emoji's
+REACTIONS = +1, -1, laugh, confused, heart, hooray, eyes
```

Comment:
Maybe add a link in comment what emojis are possible.

Reply:
@davidsvantesson good point - I like to have this too!
Reactions use the existing emojy function of gitea - i didn't wrote something special for this

Reply:
done at 69523bd592436220a77911bbdd4b4d83a7ec9e4b